File: TesteIOS/TesteIOS/Service/Entities/Event.swift
//
//  Event.swift
//  TesteIOS
//
//  Created by Levy Cristian on 23/10/20.
//

import Foundation
// MARK: - ResultElement
struct Event: Codable {
    let people: [Person]
    let date: Int
    let resultDescription: String
    let image: String
    let longitude, latitude, price: Double
    let title, id: String

    enum CodingKeys: String, CodingKey {
        case people, date
        case resultDescription = "description"
        case image, longitude, latitude, price, title, id
    }
}

typealias requestResult = [Event]

File: app/src/main/java/br/com/micaelpimentel/sicredevent/api/model/Event.kt
package br.com.micaelpimentel.sicredevent.api.model

data class Event(
    val date: Long,
    val description: String,
    val image: String,
    val longitude: Float,
    val latitude: Float,
    val price: Float,
    val title: String,
    val id: String,
    val people: List<People>,
)

data class People(
    val name: String,
    val email: String,
)